homeassistant/strings.json

```diff
@@ -68,7 +68,7 @@
     "config_flow": {
       "title": {
         "oauth2_pick_implementation": "Pick authentication method",
-        "reauth": "Reauthenticate integration",
+        "reauth": "Reauthenticate integration - {name}",
```

Comment:
We should be able to think of a better sentence that includes the name. We wouldn't want the name to be first in the sentence though, as a long name might hide what is to be repaired.

Reply:
Maybe something like: "The integration <for> {name} needs to be reauthenticated"

Not sure about the "for", it depends how other integrations use the name/title

Reply:
I've pinged UX for help :)

Reply:
I was personally thinking maybe something like

> Authentication expired for {name}

But it depends on the guidelines for repairs which I don't currently recall. Do we describe the issue or the fix the user needs to do ?